colin/core/colin.py

```diff
@@ -23,12 +23,23 @@
 logger = logging.getLogger(__name__)
 
 
-def run(target, target_type, tags=None, ruleset_name=None, ruleset_file=None,
-        ruleset=None, logging_level=logging.WARNING, checks_paths=None, pull=None,
-        insecure=False):
+def run(
+    target,
+    target_type,
+    tags=None,
+    ruleset_name=None,
+    ruleset_file=None,
+    ruleset=None,
+    logging_level=logging.WARNING,
+    checks_paths=None,
+    pull=None,
+    insecure=False,
+    skips=None,
```

Comment:
probably not an issue, but the comma after the last argument (not only here) looks strange to me

Reply:
I've tried the [black](https://github.com/ambv/black) for the reformatting. I can remove them if you wish..;-)

Reply:
I actually tend to this on my own as well: so I'm personally fine with it -- it's especially very useful when you keep adding more and you forget to add comma